src/docker-compose.yml

```diff
@@ -1,62 +1,60 @@
 services:
   anvil:
+    container_name: anvil
```

Comment:
I'm not a pro at docker, does container name have to be explicitly specified? i thought it'd inherit the service name